charts/atlantis/values.yaml

```diff
@@ -74,6 +74,15 @@ orgAllowlist: <replace-me>
 # If managing secrets outside the chart for the gitconfig, use this variable to reference the secret name
  # gitconfigSecretName: 'mygitconfigsecret'
 
+# When referencing Terraform modules in private repositories or registries (such as Artfactory)
+# configuing a .netrc file for authentication may be required:
+# netrc: |
```

Comment:
Please add these new values to the readme

https://github.com/runatlantis/helm-charts#customization